Neos.Neos/Resources/Private/Fusion/RawContentMode.fusion

```diff
@@ -13,9 +13,44 @@ prototype(Neos.Neos:RawContent) < prototype(Neos.Fusion:Template) {
 			columnContent = Neos.Neos:ContentCollection {
 				nodePath = '.'
 			}
-
 		}
 	}
+
+  prototype(Neos.Neos:ContentCase) {
+    default {
+      condition = Neos.Fusion:CanRender {
+        type = ${q(node).property('_nodeType.name')}
+      }
+    }
+    fallback {
+      @position = 'after default'
+      condition = true
+      type = 'Neos.Neos:RawContentFallbackRenderer'
+    }
```

Comment:
hacking a condition to `default` seems wierd to me, id like to have the fallback beforehand with a negated condition like:

(untested but should work)

```suggestion
    noComponentDeclared {
      @position = 'before default'
      condition = Neos.Fusion:CanRender {
        type = ${node.nodeType.name}
        @process.inverse = ${!!value} 
      }
      renderer = Neos.Neos:RawContentFallbackRenderer
    }
```

i also like `node.nodeType.name` better and using `renderer = Neos.Neos:RawContentFallbackRenderer`

Reply:
Not sure if I find this case more intuitive. I understand your idea but for me the `fallback` is the last thing to fall back to and the inverse part is easy to miss. 

With the nodetype and renderer thing I fully agree. c&p code late in the evening doesn't necessarily lead to the best Fusion implementations 😉